src/WebWindow/WebWindow.cs

```diff
@@ -83,6 +86,9 @@ public class WebWindow
         [DllImport(DllName, CallingConvention = CallingConvention.Cdecl)] static extern void WebWindow_SetTopmost(IntPtr instance, int topmost);
         [DllImport(DllName, CallingConvention = CallingConvention.Cdecl, CharSet = CharSet.Auto)] static extern void WebWindow_SetIconFile(IntPtr instance, string filename);
 
+        [DllImport(DllName, CallingConvention = CallingConvention.Cdecl, CharSet = CharSet.Auto)]
+        static extern void WebWindow_SetUriChangeCallback(IntPtr instance, UriChangeCallback callback);
```

Comment:
Could you follow the same layout as on the other lines? (i.e., single line for this, for consistency